src/resonanceReconstruction/rmatrix/legacy/unresolved/Resonance/src/ctor.hpp

```diff
@@ -0,0 +1,19 @@
+/**
+ *  @brief Constructor
+ *
+ *  @param[in] energy       the resonance energy (in eV, may be negative)
+ *  @param[in] eliminated   the level spacing energy (in eV)
+ *  @param[in] widths       the reduced widths (in sqrt(eV))
+ */
+Resonance( const Energy& energy,
+           const Energy& spacing,
+           const ReducedWidth& elastic,
+           const Width& capture,
+           const Width& fission,
+           const Width& competition ) :
```

Comment:
Do you really want to insist that these are `const`?

Reply:
Yes, because it's const correctness. I pass by const reference because I don't want to modify the values.